Services/DataX.Flow/Flow.InteractiveQueryService/PackageRoot/ServiceManifest.xml

```diff
@@ -4,7 +4,7 @@ Copyright (c) Microsoft Corporation.  All rights reserved.
 Licensed under the MIT License
 -->
 <ServiceManifest Name="Flow.InteractiveQueryServicePkg"
-                 Version="1.0.1"
+                 Version="1.0.0"
```

Comment:
curious why are we rolling back the version?

Reply:
It was mistakenly checked in. The version was rev'd was only partially for servicemanifest.xml files under dataX.Flow whereas the ApplicationManifest.xml was still having the version 1.0.0. This broke the local debugging scenario for DataX.Flow. Hence fixing the bug by reverting the version in servicemanifest.xml for each of the services since the version needs to be consistent across.